### DISCUSSION TITLE
A fatal error occurred: Could not open /dev/cu.usbserial-0001, the port doesn't exist

### DISCUSSION THREAD

**Discussion by gilnahmias** [earlephilhower/arduino-littlefs-upload#60] [Q&A] [answered]
Using Arduino IDE 2.3.2.

Following the installation instruction, I created a `/plugins` folder and placed the tool there.
<img width="364" alt="image" src="https://github.com/user-attachments/assets/7e034d10-c71f-49f1-916b-6d0ab4f267da">

I made sure to have a `/data` folder in my sketch:
<img width="257" alt="image" src="https://github.com/user-attachments/assets/4885b047-625a-4c19-913c-b737e3bb3c4a">

Under the *Tools/Board* menu I confirmed that the board is specified.

Restarted Arduino IDE 2.3.2.

Changed my board type to something else and then back again to ESP32 DEVKIT V1.

Trying to run the plugin
<img width="546" alt="image" src="https://github.com/user-attachments/assets/c14359f1-f563-42fb-853c-b9a59f00d2f0">

I get this error:
<img width="601" alt="image" src="https://github.com/user-attachments/assets/c2084644-a13a-4540-ab96-5e7bf64a67af">

What am I missing?

**Comment by earlephilhower** [comment, marked as answer]
In this case, the port specified in the IDE isn't available.  Did you close the Serial Monitor before trying to upload?  And is that the right serial port (i.e. you can upload apps with it)?

**Reply by gilnahmias** [reply]
Yes! I closed all the Serial Monitor windows (just realized that this instruction is explicitly listed) and the upload went through successfully!
